Camera/CameraAlvium/camera_alvium.cc

```diff
@@ -8,6 +8,9 @@
 #include <span>
 namespace camera
 {
+
+#define VMB_ERROR(cond) cond != VmbErrorType::VmbErrorSuccess
```

Comment:
Maybe add parenthesis around cond and the whole macro for consistency and safety